postr/instagram_postr.py

```diff
@@ -72,6 +95,16 @@ def refresh(self) -> None:
         self.followers = self._user_follower_info()
         self.followings = self._user_following_info()
 
+    @staticmethod
+    def direct_share(mediaID: str, recipients: List[int], message: str = '') -> None:
+        """
+        Shares media to a list of recipients via a direct message
+        mediaID: The id of the media to share
+        recipients: A list of the user ids to share media with
+        mesage: The message to go along with the media share
+        """
+        InstagramAPI.direct_share(mediaID, recipients, message)
+
     def spam_follower_ratio(self, uid: int = 0) -> float:
```

Comment:
why is uid not used?

Reply:
direct_share takes in a list of usernames, no uid is used. The API is inconsistent.